rules/windows/defense_evasion_sdelete_like_filename_rename.toml

```diff
@@ -0,0 +1,40 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects file name patterns as caused by the use of Sysinternals Sdelete utility to securely delete a file via multiple
+file overwrite and rename operations. Verify process details such as command line and hash to confirm this activity
+legitimacy.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Potential Secure File Deletion using SDelete utility"
+risk_score = 21
+rule_id = "5aee924b-6ceb-4633-980e-1bde8cdb40c5"
+severity = "low"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:file and event.type:change and file.name:*AAAA.AAA
```

Comment:
I'm not sure what the latest guidance is on using leading wildcards in our queries. @brokensound77, do you know the answer to this?

Reply:
`?` is okay, because it's a wildcard of only a single character, so it's more efficient than `*`

in this case, you could also do `file.extension:AAA` to avoid wildcards, until ECS has added the new wildcard type. if you can get the old extension and verify that it wasn't `AAA`, then you will probably have a pretty good balance on TP/FP

Reply:
agree, changed it to lucene rule type [event.category:file AND event.type:change AND file.name:/.+A+\.AAA/] example of file name can be found here https://github.com/elastic/detection-rules/issues/161